xmonad.cabal

```diff
@@ -84,15 +84,23 @@ library
                  , setlocale
                  , unix
                  , utf8-string           >= 0.3 && < 1.1
-  ghc-options:     -funbox-strict-fields -Wall -fno-warn-unused-do-bind
+  ghc-options:     -funbox-strict-fields -Wall -fno-warn-unused-do-bind -Wno-unused-imports
+
+  -- Keep this in sync with the oldest version in 'tested-with'
+  if impl(ghc == 8.4.3)
+     ghc-options: -Wunused-imports
```

Comment:
The oldest version we actually test with is 8.4.4 (lts-12.26), so `ghc <= 8.4.4` is probably better.
I wonder if we want to `-Werror=unused-imports` or whether we're going to check the warnings manually.

Reply:
Alright, I'll change that to `ghc <= 8.4.4`.  The `-Werror=unused-imports` bit sounds nice as well; a CI failure on one's pr is probably the best incentive possible for fixing these warnings, thanks!

Reply:
Cool.

(I wasn't sure about the -Werror bit because it's incosistent with the other warnings. But once we have reliable CI we can enable those as well. :-))